src/Module/ActivityPub/Inbox.php

```diff
@@ -61,9 +61,9 @@ protected function rawContent(array $request = [])
 			if ($owner['uid'] != $uid) {
 				throw new \Friendica\Network\HTTPException\ForbiddenException();
 			}
-			$outbox = ActivityPub\Transmitter::getInbox($uid, $page, $request['max_id'] ?? null);
+			$outbox = ActivityPub\ClientToServer::getInbox($uid, $page, $request['max_id'] ?? null);
```

Comment:
Please rename this variable to `$inbox`.

Reply:
Done